osu.Game/Beatmaps/Formats/OsuLegacyDecoder.cs

```diff
@@ -10,6 +10,7 @@
 using osu.Game.Modes;
 using osu.Game.Modes.Objects;
 using osu.Game.Screens.Play;
+using static System.Double;

```

Comment:
-1

`double.Parse` is clearer than `Parse` and it's used like twice so why bother with this